torch_geometric/nn/aggr/sort.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+from torch_geometric.nn.aggr import Aggregation
+
+
+class GlobalSortAggr(Aggregation):
```

Comment:
```suggestion
class SortAggr(Aggregation):
```
Since `Aggregation` can be used for both message passing aggregation and global pooling, let's not use `Global` inside the name. WDYT?

Reply:
Good point